Lib/email/utils.py

```diff
@@ -114,18 +114,6 @@ def getaddresses(fieldvalues):
     return a.addresslist
 
 
-
-ecre = re.compile(r'''
-  =\?                   # literal =?
-  (?P<charset>[^?]*?)   # non-greedy up to the next ? is the charset
-  \?                    # literal ?
-  (?P<encoding>[qb])    # either a "q" or a "b", case insensitive
-  \?                    # literal ?
-  (?P<atom>.*?)         # non-greedy up to the next ?= is the atom
-  \?=                   # literal ?=
-  ''', re.VERBOSE | re.IGNORECASE)
```

Comment:
This was probably left in place for backward compatibility reasons (just in case someone was using it).  Or it may have been overlooked.  No way to tell :)  But, probably better to not delete it if you backport this change.

Reply:
Since this change doesn't actual bugfix (B and Q don't have another lower cases),
I won't backport it to Python 3.6.